src/browsers/tob/shared/components/Heatmap.js

```diff
@@ -74,6 +85,22 @@ const Heatmap = ({
       .select('.domain')
       .remove()
 
+    svg
```

Comment:
Are these svg.selectAll() chains the ones we wanted to avoid in future to let react handle rendering?

Reply:
I think ideally you would define as much as you can in the DOM and set the callbacks using JSX. However, this also means more boilerplate code since we have to manually do a lot of the rendering that d3 handles, but perhaps we can make a compromise and let d3 handle drawing axes.